ops/model.py

```diff
@@ -3421,6 +3422,19 @@ def _run(
             try:
                 result = subprocess.run(args, **kwargs)  # type: ignore
             except subprocess.CalledProcessError as e:
+                if 'access denied' in e.stderr.lower():
```

Comment:
Are we sure this is the only text from Juju that represents an auth failure?

Reply:
I think so, but I'll verify before moving this out of draft, yes.